docker-compose.yml

```diff
@@ -1,45 +1,45 @@
-# To use this file, run `docker-compose up`.
-version: '3'
-
+version: '2'
```

Comment:
It looks like the work  in `develop` made these changes to the `docker-compose.yml` file obsolete.